### DISCUSSION TITLE
Fallback to wifi AP when connecting fails

### DISCUSSION THREAD

**Discussion by jeroen79** [jeroen79/esp-brew-engine#16] [Ideas]
When connecting fails it might be a good idea to just switch to AP mode, then at least the user can connect and correct wifi settings

**Comment by jonsenior79** [comment]
I'm guessing from the behaviour with version 1.6.0 that this hasn't been implemented.

I have two networks, one services the house and the other services the shed (where my brewing activites take place). When I install the firmware I can connect via the AP and setup the wifi connection to use the house network. The device then connects to the house network and can be accessed via a browser. When I change the wifi credentials to those for the shed and move the device to the shed I can see it on the local wifi router (configured as an AP with no DHCP and wired to the main network in the house), but it keeps dropping the connection before managing to acquire an IP address. Powering it up in the house where it cannot reach the shed network it doesn't seem to create the default AP so at this point the only way to regain control of the unit is to reload the firmware.

_Edit to add: In addition, the scan button doesn't seem to work. Or at least it fails to find any wifi networks whereas there are at least two in the house and one in the shed. I'm using a "standard" 30-pin WROOM ESP32 which is wired up to an SSR and a DS18B20._

**Comment by jeroen79** [comment]
Try lowering the wifi power, some eps32 board have issues when its all set to high
